src/signals_notebook/users/group.py

```diff
@@ -0,0 +1,225 @@
+        api = SignalsNotebookApi.get_default_api()
+        log.debug('Create User: %s...', cls.__name__)
+
+        response = api.call(
+            method='POST',
+            path=(cls._get_endpoint(),),
+            json={
+                'data': {
+                    'attributes': request.dict(by_alias=True),
+                }
+            },
+        )
+        log.debug('Group: %s was created.', cls.__name__)
+
+        result = GroupResponse(**response.json())
+        return cast(ResponseData, result.data).body
+
+    def save(self, force: bool = True) -> None:
+        """Update attributes of a specified group.
+
+        Returns:
+
+        """
+        api = SignalsNotebookApi.get_default_api()
+        api.call(
+            method='PATCH',
+            path=(
+                self._get_endpoint(),
+                self.id,
+            ),
+            params={
+                'force': json.dumps(force),
+            },
+            json={
+                'data': {
+                    'attributes': self.dict(
+                        include={
+                            'name',
+                            'description',
+                        },
+                    )
+                },
+            },
+        )
+
+    def delete(self):
+        """Delete user group by id.
+
+        Returns:
+
+        """
+        api = SignalsNotebookApi.get_default_api()
+        log.debug('Disable Group: %s...', self.id)
+
+        api.call(
+            method='DELETE',
+            path=(self._get_endpoint(), self.id),
+        )
+        log.debug('Group: %s was disabled successfully', self.id)
+
+    def get_members(self) -> list[GroupMember]:
+        """Get user group members
+
+        Returns:
+            Group members
+        """
+        api = SignalsNotebookApi.get_default_api()
+        response = api.call(
+            method='GET',
+            path=(self._get_endpoint(), self.id, 'members'),
+        )
+        result = GroupMemberResponse(**response.json())
+        return [cast(ResponseData, item).body for item in result.data]
+
+    def add_user(self, user: User, force: bool = True) -> list[GroupMember]:
+        """Add user to user group
+
+        Args:
+            user: User object
+            force: Force to update
+
+        Returns:
+            list[GroupMember]
+        """
+        api = SignalsNotebookApi.get_default_api()
+
+        response = api.call(
+            method='POST',
+            path=(self._get_endpoint(), self.id, 'members'),
+            params={
+                'force': json.dumps(force),
+            },
+            json={
+                'data': {
+                    'attributes': {
+                        'userId': user.id
+                    },
+                }
+            },
+        )
+
+        result = GroupMemberResponse(**response.json())
+        return [cast(ResponseData, item).body for item in result.data]
+
+    def delete_user(self, user: User) -> None:
```

Comment:
and this method will be useless because we will create _reload_members method for lazy load

Reply:
Its really useful to get members to class attribute. It will be awesome. CRUD operations with members will be easier